src/main/java/org/gridsuite/study/server/NetworkModificationService.java

```diff
@@ -0,0 +1,211 @@
+                .queryParam("group", groupUuid)
+                .buildAndExpand()
+                    .toUriString();
+
+            return webClient.put()
+                    .uri(getNetworkModificationServerURI(true) + path)
+                    .body(BodyInserters.fromValue(groovyScript))
+                    .retrieve()
+                    .bodyToFlux(new ParameterizedTypeReference<ElementaryAttributeModificationInfos>() {
+                    });
+        });
+    }
+
+    Flux<ElementaryAttributeModificationInfos> applyLineChanges(UUID studyUuid, String lineId, String status, UUID groupUuid) {
+        Objects.requireNonNull(studyUuid);
+        Objects.requireNonNull(lineId);
+        return networkStoreService.getNetworkUuid(studyUuid).flatMapMany(networkUuid -> {
+            var path = UriComponentsBuilder.fromPath(buildPathFrom(networkUuid) + "lines" + DELIMITER + "{lineId}" + DELIMITER + "status")
+                .queryParam("group", groupUuid)
+                .buildAndExpand(lineId)
+                    .toUriString();
+
+            return webClient.put()
+                    .uri(getNetworkModificationServerURI(true) + path)
+                    .body(BodyInserters.fromValue(status))
+                    .retrieve()
+                    .onStatus(httpStatus -> httpStatus != HttpStatus.OK, this::handleChangeLineError)
+                    .bodyToFlux(new ParameterizedTypeReference<ElementaryAttributeModificationInfos>() {
+                    });
+        });
+    }
+
+    private Mono<? extends Throwable> handleChangeLineError(ClientResponse clientResponse) {
+        return clientResponse.bodyToMono(String.class).flatMap(body -> {
+            String message = null;
+            try {
+                JsonNode node = new ObjectMapper().readTree(body).path("message");
+                if (!node.isMissingNode()) {
+                    message = node.asText();
+                }
+            } catch (JsonProcessingException e) {
+                if (!body.isEmpty()) {
+                    message = body;
+                }
+            }
+            return Mono.error(new StudyException(LINE_MODIFICATION_FAILED, message));
+        });
+    }
+
+    public Flux<ElementaryModificationInfos> createLoad(UUID studyUuid, String createLoadAttributes, UUID groupUuid) {
+        Objects.requireNonNull(studyUuid);
+        Objects.requireNonNull(createLoadAttributes);
+        return networkStoreService.getNetworkUuid(studyUuid).flatMapMany(networkUuid -> {
+            var path = UriComponentsBuilder.fromPath(buildPathFrom(networkUuid) + "createLoad")
+                .queryParam("group", groupUuid)
+                .buildAndExpand()
+                .toUriString();
+
+            return webClient.put()
+                .uri(getNetworkModificationServerURI(true) + path)
+                .contentType(MediaType.APPLICATION_JSON)
+                .body(BodyInserters.fromValue(createLoadAttributes))
```

Comment:
Need `                    .onStatus(httpStatus -> httpStatus == HttpStatus.NOT_FOUND, clientResponse -> Mono.error(new StudyException(ELEMENT_NOT_FOUND)))
`
because the all 404 from net-work-modification-server are are transcribed by 500 errors in the study server

Reply:
Done